lib/rules/new-cap.js

```diff
@@ -144,6 +144,22 @@ module.exports = function(context) {
         return false;
     }
 
+    /**
+     * Reports the given message for the given node. The location will be the start of the property or the callee.
+     * @param {ASTNode} node CallExpression or NewExpression node.
+     * @param {string} message The message to report.
+     * @returns {void}
+     */
+    function report(node, message) {
+        var callee = node.callee;
+
+        if (callee.type === "MemberExpression") {
+            callee = callee.property;
+        }
+
+        context.report(node, callee.loc.start, message);
```

Comment:
Formatters can use those types as well. I think, for this rule, it makes more sense to have the type of a `NewExpression` or `CallExpression` reported.